distribution/pom.xml

```diff
@@ -72,7 +72,7 @@
                             <includeGroupIds>${project.groupId}</includeGroupIds>
                             <includeArtifactIds>hibernate-ogm-modules</includeArtifactIds>
                             <includeTypes>zip</includeTypes>
-                            <includeClassifiers>wildfly-11-dist</includeClassifiers>
+                            <includeClassifiers>wildfly-12-dist</includeClassifiers>
```

Comment:
There is a property with this value, can't we use it? It's called hibernateWildflyClassifier